tests/components/technove/test_init.py

```diff
@@ -0,0 +1,15 @@
+"""Tests for the TechnoVE integration."""
+
+
+from homeassistant.components.technove.const import DOMAIN
+from homeassistant.core import HomeAssistant
+
+from tests.common import MockConfigEntry
+
+
+async def test_setting_unique_id(
```

Comment:
Maybe you can replace this test with a test where you test loading and unloading

Reply:
Done.